lib/zht/pages/ActualPosition.dart

```diff
@@ -38,8 +40,7 @@ class ActualPositionState extends State<ActualPosition>{
     }
     
     _locationData = await location.getLocation();
-    setState(() {    
```

Comment:
why did you removed this part ? The current location of the user is no longer visible without this code.